### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following: The GitHub account you are using was recently created, and you appear to be having difficulty demonstrating your academic affiliation. Please wait a few days then try again.

### DISCUSSION THREAD

**Discussion by amreahmed** [community/community#79162] [GitHub Education]
### Hi everyone,

I can't verify that i am a student with my university ID. i tried like 100 times, and every time i got the error that my first name is not the same in billing information, but it's the same and then i got this error 

Your application cannot be reviewed until you fix the following: The GitHub account you are using was recently created, and you appear to be having difficulty demonstrating your academic affiliation. Please wait a few days, then try again.

can any one from support help please

**Comment by VictorMoni** [comment]
Hi @amreahmed,  😃👍

I know this can be frustrating. From the error message, it looks like there are **two separate issues**:  

1. **Name mismatch** → Even small differences (extra spaces, capital letters, middle names) between your GitHub profile and your university document can cause rejection. Try editing your GitHub profile name to exactly match your student ID.  

2. **New account limitation** → GitHub often rejects very new accounts. In this case, you’ll need to wait a few days, keep your account active (commits, repos, contributions), and then reapply.  

If the issue continues, the best option is to **open a ticket with GitHub Education Support** and attach your enrollment proof: 
 
👉 [https://support.github.com/contact](https://support.github.com/contact)  

This way the team can manually review your application.  

Good luck — hope you get approved soon! 🚀

**Comment by nafez-07** [comment]
Hi,

I had the same issue **_The GitHub account you are using was recently created, and you appear to be having difficulty demonstrating your academic affiliation. Please wait a few days then try again._**

After changing my GitHub username to match the name on my academic document, the issue was resolved.
